CHANGELOG.md

```diff
@@ -4,18 +4,25 @@ Changelog
 Development version
 -------------------
 
+The highlight of this release is the support for Shapely 2.0. This makes it possible to
+test Shapely 2.0 (currently a first beta is released) alongside GeoPandas.
+
+Note that if you also have PyGEOS installed, you need to set an environment variable
+(`USE_PYGEOS=0`) before import geopandas to actually test Shapely 2.0. See
```

Comment:
```suggestion
(`USE_PYGEOS=0`) before import geopandas to actually test Shapely 2.0 features instead of PyGEOS. See
```

(this part could potentially be a little confusing since you'd be using the core Shapely features either way, but you'd need to opt-in to the vectorized Shapely 2.0 features)